libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -332,11 +333,25 @@ proc validateAndRelay(g: GossipSub,
       g.floodsub.withValue(t, peers): toSendPeers.incl(peers[])
       g.mesh.withValue(t, peers): toSendPeers.incl(peers[])
 
+
     # Don't send it to source peer, or peers that
     # sent it during validation
     toSendPeers.excl(peer)
     toSendPeers.excl(seenPeers)
 
+    if msg.data.len > msgId.len * 10:
```

Comment:
Can we make this condition more descriptive - like making it a proc or adding a comment with the reasoning?